assets/check

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import boto3
+import botocore
+import json
+import re
+import sys
+from packaging import version as semver
+
+DEFAULT_AWS_REGION = "eu-west-2"
+DEFAULT_VER_REGEX = "(\d{1,3}\.\d{1,3}\.\d{1,3})$"
+REQ_PARAMS = [
+    "aws_access_key_id",
+    "aws_secret_access_key",
+    "aws_region",
+    "ami_prefix",
+    "version_regex"
+]
+
+
+def validate_source(payload):
```

Comment:
I'd be tempted to rename this to something like `process_source` or `process_payload` as it may actively modify the content rather than just straight up validation. Or perhaps split the default seeding aspects into one function and the validation into another?

Reply:
Updated in [5d8978c](https://github.com/companieshouse/ci-resource-type-ami-id/pull/1/commits/5d8978c101a0a24820850c2beb64d8461f6cd67e)